### ISSUE TITLE
Not able to install this project

### ISSUE THREAD

**Issue by sudhakar410** [alexanderson1993/space-edventures#120]
I am not able to install this project. I am getting lots of issues in packages. Is this project active or working still ?

**Comment by alexanderson1993** [comment]
I mean, the last commit was 3 years ago, I made this for a capstone project. The app itself is up and running at spaceedventures.org, but I'm not really in a place to make sure the codebase itself is up to date.

If you can share specific details about the problems you're having, I might be able to help.

**Comment by sudhakar410** [comment]
hi,
npm ERR! No versions available for gatsby-source-firebase-firestore

Thanks

**Comment by alexanderson1993** [comment]
Looks like it changed to `gatsby-source-firestore`. You'll have to set it up with your own Firebase credentials.

**Comment by sudhakar410** [comment]
![space-centerissue](https://user-images.githubusercontent.com/53808329/199744168-a779cce8-7b4d-4afc-a02e-4037ecc6de55.PNG)

please find attached issue

**Comment by alexanderson1993** [comment]
Yeah, the project is super out of date. You'll have to figure out how to get around those issues.
